src/gt4py/next/embedded/operators.py

```diff
@@ -188,7 +188,9 @@ def _tuple_at(
     @utils.tree_map
     def impl(field: common.Field | core_defs.Scalar) -> core_defs.Scalar:
         res = field[pos] if common.is_field(field) else field
-        res = res.item() if hasattr(res, "item") else res  # extract scalar value from array
+        res = (
+            res.ndarray.item() if hasattr(res, "ndarray") else res  # type: ignore[union-attr]
+        )  # extract scalar value from array
```

Comment:
I think we should enable something like 
```
res = field[pos].item() if common.is_field(field) else field
```
or `.asscalar()` as we previously discussed. This is different from having the built-in function.

Reply:
```python
field[pos].ndarray.asscalar()
Python 3.10.0 (v3.10.0:b494f5935c, Oct  4 2021, 14:59:20) [Clang 12.0.5 (clang-1205.0.22.11)]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.12.3 -- An enhanced Interactive Python. Type '?' for help.
PyDev console: using IPython 8.12.3
Traceback (most recent call last):
  File "/Users/nicolettafarabullini/PycharmProjects/gt4py/.venv/lib/python3.10/site-packages/IPython/core/interactiveshell.py", line 3508, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-1-0aa42db6c80a>", line 1, in <module>
    field[pos].ndarray.asscalar()
AttributeError: 'numpy.ndarray' object has no attribute 'asscalar'
```

Reply:
@havogt  means that `asscalar` or `as_scalar` should be a new member function of Field doing conceptually the same  as `ndarray.item()`.

Reply:
yes